npm-js/create-kalix-entity/template/event-sourced-entity/proto/counter_domain.proto

```diff
@@ -4,15 +4,6 @@ package com.example.domain;
 
 import "kalix/annotations.proto";
```

Comment:
Should be possible to remove the kalix annotations import from the domain files, no longer used.

Reply:
done